app/controllers/events_controller.rb

```diff
@@ -11,17 +10,15 @@ def show
   def new
     @event = Event.new
```

Comment:
This action should be responsible for listing events created by the user.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)). 

- To fulfil this, we can change this to 👇 

```suggestion
    @created_events = current_user.events
```

Reply:
## Correction: Wrong suggested changes

- The earlier suggested changes were meant for the `show` method and not the `new` method. Apologies. 🙏